Mailer/README.md

```diff
@@ -887,27 +1176,158 @@ new *email* template via API.
 | --- |---| --- | --- |
 | Authorization | Bearer *String* | yes | API token. |
 
+##### *Body:*
+
+```json5
+{
+  // optional
+  "public_listing": true // Boolean; Get only published newsletters
+}
+```
+
 ##### *Example:*
 
 ```shell
-curl -X GET \
+curl -X POST \
   http://mailer.remp.press/api/v1/mailers/mail-types \
-  -H 'Authorization: Bearer XXX'
+  -H 'Authorization: Bearer XXX' \
+  -H 'Content-Type: application/json' \
+  -d '{
+    "public_listing": true
+}'
 ```
 
 Response:
 
 ```json5
 {
-    "status": "ok",
-    "data": [
-        {
-            "id": 14,
-            "code": "alerts",
-            "title": "Breaking news"
-        }
-        // ...
-    ]
+  "2": {
```

Comment:
Please maintain response format backwards compatible. You cannot change existing APIs as you'd break existing client implementations.